lib/instrumentation/core/http-outbound.js

```diff
@@ -227,6 +235,8 @@ function applySegment(opts, makeRequest, hostname, segment) {
       segment.addSpanAttribute(`request.parameters.${key}`, parsed.parameters[key])
     }
   }
+  segment.attributes.addAttribute(DESTINATIONS.SPAN_EVENT, 'host', host)
```

Comment:
adding destination of span event as we don't want these attrs on the segment. the `lib/spans/span-event` and `streaming-span-event` will take those and make `server.address` and `server.port`.  I plan on filling a follow up to encapsulate all the external segment/span attributes into a function on the segment so we can call `captureExternalAttributes`